qemu/tests/discard_for_raw_block_target.py

```diff
@@ -0,0 +1,56 @@
+import logging
+import os
+
+from provider.qemu_img_utils import strace
+
+from avocado import fail_on
+from avocado.utils import process
+
+from virttest import data_dir
+from virttest.qemu_storage import QemuImg
+
+
+def run(test, params, env):
```

Comment:
If you don't need to get a VM environment, what does env mean here?

Reply:
I agree. I will try with it. Thanks.

Reply:
After trying, case can not work with 2 arguments as below. 

```ERROR: run() takes 2 positional arguments but 3 were given```

Reply:
Sorry my bad, because process() need avocado-vt/virttest/env_process.py.
This is a bad idea, please ignore.